Image component "./image.types" not found

**Describe the bug**
I can't actually build the component library. Whenever I try to, I get an error message stating 

``` Plugin typescript: @rollup/plugin-typescript TS2307: Cannot find module './image.types' or its corresponding type declarations.
src/image/image.tsx: (2:28)
```

**To Reproduce**
Steps to reproduce the behavior:
1. Download the application
2. Run the start command `pnpm run develop` 
3. Look at the console 
4. See error

**Expected behavior**
I should be able to run this application without any issues.  

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Desktop (please complete the following information):**
 - OS: Windows 
 - Browser: Firefox
 - Version: 125.0.1

**Additional context**
I'll be happy to send in a quick PR to fix this.

--- a/src/image/image.tsx
+++ b/src/image/image.tsx
@@ -1,5 +1,5 @@
 import React from "react";
-import { ImageProps } from "./image.types";
+import { type ImageProps } from "./types";
 
 export const Image = React.forwardRef<HTMLImageElement, ImageProps>(
 	({ alt, src, responsive, className, ...props }, ref): JSX.Element => {

